configuration/developing-community-health-applications.md

```diff
@@ -520,125 +529,122 @@ To build your targets into your app, you must compile them into app-settings, th
     medic-conf --local compile-app-settings backup-app-settings upload-app-settings 
 
 
-## Contacts
-In the `app_settings.contact_summary` you can write a script to output fields for the contact info pane and help decide which reports should be able to be filed against a contact. The script is evaluated as JavaScript so all the standard language features are available. To make this easier to write and maintain, a declarative format is also available. In the `contact_summary.templated.js` file you specify three variables: `context`, `fields`, and `cards`.
+## Contact Summaries
```

Comment:
Even though this code is stored in `app_settings.contact_summary`, and the file for it is `contact-summary.templated.js`, I don't think _Contact Summaries_ defines this section well because it affects the context for the actions as well. What about _Contact Pages_, or _Contact Profiles_?

We need to remember to update the index as well if we change this from _Contacts_.

Reply:
I'm fine just reverting back to Contacts.